apps/microsoft-teams/msteams-bot-service/src/services/ms-teams-conversation-service.ts

```diff
@@ -0,0 +1,36 @@
+import { BotBuilderCloudAdapter } from '@microsoft/teamsfx';
+import ConversationBot = BotBuilderCloudAdapter.ConversationBot;
+import { Request, Response } from 'express';
+import { TurnContext } from 'botbuilder';
+
+export class MsTeamsConversationService {
+  constructor(private readonly msTeamsConversationBot: ConversationBot) {}
+
+  static fromBotCredentials(botId: string, botPassword: string): MsTeamsConversationService {
+    const conversationBot = new ConversationBot({
+      adapterConfig: {
+        MicrosoftAppId: botId,
+        MicrosoftAppPassword: botPassword,
+        MicrosoftAppType: 'MultiTenant',
+      },
+      notification: {
+        enabled: true,
+      },
+    });
```

Comment:
Nit: What are your thoughts about moving this config into a separate file?

Reply:
Yeah, I played with exactly this idea and think it makes a lot of sense. We could eg initialize these directly in config and then pass them through vs. only initializing the env vars. I suggest we leave open for a future iteration, wdyt?